### DISCUSSION TITLE
iOS Privacy Manifest

### DISCUSSION THREAD

**Discussion by raul-saez** [invertase/react-native-firebase#7657] [General]
Hey all 👋🏾 

Looking at the AppStore [Upcoming third-party SDK requirements](https://developer.apple.com/support/third-party-SDK-requirements/) there are a bunch of Firebase's SDKs listed as they require a privacy manifests and signatures.

Firebase for iOS seems to support this already from [v10.21.0 ](https://firebase.google.com/support/release-notes/ios#version_10210_-_february_6_2024).

How does this affect react-native-firebase?

**Comment by mikehardy** [comment]
Hey @raul-saez - it affects react-native-firebase exactly as specified in our CHANGELOG, we have just released an update with firebase-ios-sdk 10.21.0 and are paying close attention so we can release the updated firebase-ios-sdk that actually has the manifests, sometime next week

https://github.com/invertase/react-native-firebase/blob/main/CHANGELOG.md#1900-2024-02-26
